peachjam/adapters/adapters.py

```diff
@@ -157,7 +157,7 @@ def update_document(self, url):
         model = self.get_model(document)
         logger.info(f"Importing as {model}")
 
-        if hasattr(model, "nature"):
+        if hasattr(model, "nature") and document["subtype"]:
             document_nature_name = " ".join(
```

Comment:
@longhotsummer I added this line because it seems some documents come in without the subtype property

Reply:
The model now will always have `nature`, so that can be skipped. Guarding against a subtype makes sense.